packages/flutter_tools/lib/src/test/flutter_platform.dart

```diff
@@ -492,6 +513,9 @@ void main() {
     }
     if (host.type == InternetAddressType.IP_V6)
       command.add('--ipv6');
+    if (bundlePath != null) {
+      command.add('--flutter-assets-dir=$bundlePath');
```

Comment:
@aam Do you recall why this was needed?

It is a bit odd that we are passing the SDK path as the asset dir.

Reply:
This implies all assets to be used by tester needs to be put in the sdk path. That's quite odd. In platform packaging the dill files are not found in the asset directory. Would it make sense to separate these concerns and change the way the dill files are located in the tester? [probably a new flag]?

Reply:
Yeah, it does make sense.

Reply:
@tvolkert FYI I will attempt this if you don't have any objection. The request is to change tester to locate dill files w/o needing the asset flag so we can safely remove this particular line of code in flutter_platform.dart. When we add proper asset support [a separate param to installHook?], we can add it back. If you disagree with the methodology, let me know.